ietf/person/management/commands/deactivate_email_addresses.py

```diff
@@ -84,7 +84,7 @@ def handle(self, *args, **options):
                     email.origin = email.person.user.username if email.person.user_id else ('script: %s deactivation' % options['reason'])
                     email.save()
                     PersonEvent.objects.create(person=email.person, type='email_address_deactivated',
-                        desc="Deactivated the email addres <%s>. Reason: %s" % (email.address, options['reason']) )
+                        desc="Deactivated the email address <%s>. Reason: %s" % (email.address, options['reason']) )
```

Comment:
Several steps.
* `ietf/manage.py makemigrations --empty --name=personevent_spelling person
* Add a forward and reverse function to the empty migration it creates and wire them into `migrations.RunPython` (search for `def forward(` to find lots of examples.
* update the dependencies list to the tip of any apps from which you end up pulling in the model. (You'll at least pull in PersonEvent from Person, using `apps.get_model`, not an import, but you'll get the right person dependency automatically given that you made the migration on the person app. Note also that what you get is a facade with a working object manager, not the actual class - member functions of the class will not be available).
* the reverse should just `pass`. For the forward, you'll want to figure out how to iterate over the objects that have the spelling errors and update them as efficiently as you can (since however long this runs is downtime as the release it contains is deployed).

More to read at https://docs.djangoproject.com/en/2.2/topics/migrations/

Reply:
I think this is beyond my paygrade...